src/components/CreateFactorySteps.vue

```diff
@@ -418,12 +418,12 @@ export default createComponent({
       date: (new Date(Date.now() - (new Date()).getTimezoneOffset() * 60000)).toISOString().substr(0, 10),
       /** locale time */
       time: (new Date(Date.now() - (new Date()).getTimezoneOffset() * 60000)).toISOString().substring(11, 16),
-      bearNumber: 0,
-      bearType: 0,
-      bearSize: 0,
-      bearSizeNumber: 0,
-      bearSex: 0,
-      bearFeature: '',
+      bearNumber: 1,
+      bearType: [],
+      bearSize: [],
+      bearSizeNumber: [],
+      bearSex: [],
+      bearFeature: [],
```

Comment:
Interesting...thanks for the extension. I created an issue for backend to be aware of these new data fields https://github.com/tai271828/disfactory-backend/issues/21